src/reducers/entities/search.js

```diff
@@ -13,7 +13,7 @@ function results(state = [], action) {
         }
         return action.data.order;
     }
-    case PostTypes.REMOVE_POST: {
+    case PostTypes.POST_REMOVED: {
```

Comment:
This would require us to make changes to webapp as well as we are using them `PostTypes.REMOVE_POST` action type

Reply:
Already done :)

https://github.com/mattermost/mattermost-webapp/pull/2343/files#diff-352a5ff733bbce75800e45b15f77ee14R305